bin/cron/pardot_sync

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/env ruby
```

Comment:
Per our offline discussion, we will pursue this in a future PR. We need some further thinking on what the right thing to do here. My thinking at the moment is when we are aware of multiple accounts being used by the same person (via the data in the user_profiles table), we maintain both contacts separately. This means we potentially email both contacts. But that may be the right answer - we know there are teachers that use their school account during the school year, but don't have access to it over the summer and use their personal account in the summer. But we may update properties via the association so we mark the teacher account as having been through PD, for instance.